test/tests/server.js

```diff
@@ -339,6 +339,7 @@ describe('Server', () => {
       request(app).get('/foo/bar')
         .expect(200, () => {
           assert(locals.webpackStats);
+          assert(locals.fs);
```

Comment:
> we need add test to avoid removing this in future (accidentally)
@evilebottnawi: it was already added. See here

Reply:
Oh sorry